devops/azure-pipelines/azure-pipelines.yml

```diff
@@ -20,24 +25,29 @@ stages:
           sonarProjectName: 'Forge-Backend'
           testPath: '**/surefire-reports/TEST-*.xml'
           mavenOutput: demo-0.0.1-SNAPSHOT.war
+          
   - stage: Dockerize
     jobs:
       - template: ./dockerize-template.yml
         parameters:
           jobName: 'Dockerize'
           containerRegistry: 'acr-team2'
           repository: 'forge-backend'
-          imageTag: "$(Build.SourceVersion)"
+          imageTag: latest
           dockerPath: '**/mavenpipe.dockerfile'
+
   - stage: Deploy
```

Comment:
isn't this deploying PR builds as well as other builds now that you took out the condition from the k8s template? not good. the condition probably belongs out here anyway (as conditional insertion)

Reply:
oh. shoot.

Reply:
Deploy stage will no longer run unless on a CI build on either main or dev